posthog/session_recordings/test/test_lts_session_recordings.py

```diff
@@ -20,12 +20,16 @@ def setUp(self):
         self.team = Team.objects.create(organization=self.organization, name="New Team")
 
     @patch(
-        "posthog.session_recordings.queries.session_replay_events.SessionReplayEvents.exists",
+        "posthog.session_recordings.queries_to_replace.session_replay_events.SessionReplayEvents.exists",
+        return_value=True,
+    )
+    @patch(
+        "posthog.session_recordings.queries_to_delete.session_replay_events.SessionReplayEvents.exists",
```

Comment:
duplicating the patches to force me to consider them properly when we remove the old folder